j5/backends/hardware/zoloto/camera_board.py

```diff
@@ -0,0 +1,100 @@
+"""Hardware implementation of the Zoloto Virtual Camera Board."""
+
+from os.path import exists
+from pathlib import Path, PosixPath
+from platform import system
+from typing import Optional, Set, Type, TypeVar
+
+from zoloto import __version__ as zoloto_version
+from zoloto.cameras.camera import Camera
+
+from j5.backends import Backend
+from j5.backends.hardware import HardwareEnvironment
+from j5.boards import Board
+from j5.boards.zoloto import ZolotoCameraBoard
+from j5.components import MarkerCameraInterface
+from j5.vision import Coordinate, Marker, MarkerList
+
+CAMERA_PATH = PosixPath("/dev/video0")
+CAMERA_SERIAL = "video0"
+
+T = TypeVar("T", bound=Camera)
```

Comment:
Is 'T' left over from development or does it serve a purpose?

Reply:
It's used in the backend constructor to allow subclasses of Camera